dashboard/dashboard-session-sso.md

```diff
@@ -149,3 +149,49 @@ TiDB Dashboard 支持基于 [OIDC](https://openid.net/connect/) 协议的单点
     ![操作示例 3](/media/dashboard/dashboard-session-sso-okta-info-3.png)
 
 至此，TiDB Dashboard 已被配置为使用 Okta 进行 SSO 登录。
+
+## 示例二：使用 Auth0 进行 TiDB Dashboard SSO 登录认证
+
+和 Okta 类似，[Auth0](https://www.okta.com/) 也可以提供 OIDC SSO 的身份认证服务。
```

Comment:
It seems the link to Auth0 is not correct. The current link is https://www.okta.com/ and should be changed to https://auth0.com/. PTAL. @baurine

Reply:
Good catch, let me fix it soon.